src/systemcmds/param/param.cpp

```diff
@@ -561,11 +568,13 @@ do_save_default()
 }
 
 static int
-do_show(const char *search_string, bool only_changed)
+do_show(const char *search_string, bool only_changed, bool only_used)
 {
+	if (only_used) {
+		PARAM_PRINT("Warning! Only parameters used during runtime will be shown\n");
```

Comment:
A note should be enough.
```suggestion
		PARAM_PRINT("Note: only parameters used during runtime will be shown\n");
```